src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md

```diff
@@ -0,0 +1,63 @@
+title=Hawkular Metrics 0.21.0 - Release
+author=Stefan Negrea
+date=2016-10-25
+type=post
+tags=blog, metrics, release
+status=published
+~~~~~~
+
+I am happy to announce release 0.21.0 of Hawkular Metrics.This release is anchored by performance enhancements and general fixes.
```

Comment:
We do not announce patch release and the patch releases so far only have fixes for OS specific issues. There are no changes to the general distribution.